src/ansi.rs

```diff
@@ -1516,11 +1548,13 @@ where
             },
             ('L', []) => handler.insert_blank_lines(next_param_or(1) as usize),
             ('l', intermediates) => {
+                let is_private = intermediates.first() == Some(&b'?');
```

Comment:
Wouldn't `('l', [b'?', ...])` work here? That way we could just use two separate match statements instead of having to create a separate conditional.

Reply:
it'll work just fine.